patches/tModLoader/Terraria/ModLoader/GlobalItem.cs

```diff
@@ -686,6 +686,17 @@ public abstract class GlobalItem : GlobalType<Item, GlobalItem>
 			return true;
 		}
 
+		/// <summary>
+		/// Allows you to make things happen when items stack together.
+		/// </summary>
+		/// <param name="item1">The item that will be removed or have its stack reduced.</param>
+		/// <param name="item2">The item that will have its stack increased.</param>
+		/// <param name="numberToBeTransfered">The number to be transfered from item1 to item2.</param>
+		/// <param name="firstStackSplit">Only true when right clicking an item when the mouse item is air.  The clicked item is cloned and stack set to 0.</param>
+		public virtual void OnStack(Item item1, Item item2, int numberToBeTransfered, bool firstStackSplit) {
```

Comment:
This documentation is what I would expect, but we should call them `from` and `to` or `source` and `dest` to make it very clear which is which.

Reply:
changed them to increase and decrease